conf/modules.config

```diff
@@ -644,6 +644,7 @@ process{
         ext.prefix       = { meta.num_intervals <= 1 ? ( params.joint_germline ? "${meta.id}.haplotypecaller.g" : "${meta.id}.haplotypecaller" ) : ( params.joint_germline ? "${meta.id}.haplotypecaller.${intervals.simpleName}.g" :"${meta.id}.haplotypecaller.${intervals.simpleName}" ) }
         ext.when         = { params.tools && params.tools.split(',').contains('haplotypecaller') }
         publishDir       = [
+            enabled: !params.joint_germline,
```

Comment:
I think we should publish haplotypecaller results also when joint germline is enabled so users can investgate the vcfs also before joint germline calling

Reply:
these are the `.g.vcf.gz` intermediate files.

Reply:
sure but also for the other tools that do some type of filtering after variantcalling, we publish the "raw" vcfs.

Reply:
you want to publish the 133 g.vcf.gz files per sample?

Reply:
ed after discussion on gather town. gVCFs are published by merge_vcfs in joint germlie subworkflow